packages/gatsby-core-utils/src/fetch-remote-file.ts

```diff
@@ -236,14 +229,12 @@ async function fetchFile({
       await fs.remove(tmpFilename)
     }
 
-    if (IS_WORKER) {
-      await cache.set(cacheIdForWorkers(url), {
-        status: `complete`,
-        result: filename,
-        workerId: WORKER_ID,
-        buildId: BUILD_ID,
-      })
-    }
+    await cache.set(cacheIdForWorkers(url), {
+      status: `complete`,
+      result: filename,
+      workerId: WORKER_ID,
+      buildId: BUILD_ID,
+    })
 
     return filename
   } catch (err) {
```

Comment:
Should we also remove `if (IS_WORKER)` from the catch block?

Reply:
Ah, yep

Reply:
Adjusted in https://github.com/gatsbyjs/gatsby/pull/33932/commits/d98f51095907d44e2622a3c1851c5911ee0bf645